lib/ui5Framework/AbstractResolver.js

```diff
@@ -2,7 +2,13 @@ const path = require("path");
 const log = require("@ui5/logger").getLogger("ui5Framework:AbstractResolver");
 const semver = require("semver");
 
-const versionRegExp = /^(0|[1-9]\d*)\.(0|[1-9]\d*)(?:\.(0|[1-9]\d*))?$/;
+const VERSION_RANGE_REGEXP = new RegExp(
```

Comment:
Good point. I've added comments a test to check against the pattern from the JSON schema.

We might even just take the pattern from the JSON here instead of having a copy. What do you think?